object_detection_node:
  ros__parameters:
    standalone: true
    # minimum confidence probability for a detection to be accepted
    minimum_prob: 0.4
    

    # topic names
    #raw image frames from the drone's camera
    image_raw_topic : "/camera/image_raw"
    all_detected_topic : "/all_detected" # image frames in which all persons (and specified objects) are detected
    bounding_boxes_topic : "/all_bounding_boxes" # list of person bounding boxes
    tracking_info_topic : "/tracking_info" # topic to publish the information on persons and objects for llm-base commands package

    
associator_node:
  ros__parameters:
    standalone: true


